File: JobQuestFrontend/src/app/models/appliedJobs/applied-job-info.model.ts
export class AppliedJobInfoModel {
  appliedJobId?: number;
  id: number = 0;
  jobId: number = 0;
  compId: number = 0;
  jobName: string = "";
	isJobRejected: boolean = false;

  constructor(
    appliedJobId: number = 0,
    id: number = 0,
    jobId: number = 0,
    compId: number = 0,
    jobName: string = "",
    isJobReject: boolean = false
  ) {
    this.appliedJobId = appliedJobId;
    this.id = id;
    this.jobId = jobId;
    this.compId = compId;
    this.jobName = jobName;
    this.isJobRejected = isJobReject;
  }
}

File: JobQuestRestBackend/src/main/java/com/cybage/boot/models/AppliedJobInfoModel.java
package com.cybage.boot.models;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "AppliedJobDetails")
public class AppliedJobInfoModel {

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	int appliedJobId;
	private int id;
	private int jobId;
	private int compId;
	private String jobName;
	private Boolean isJobRejected;
	
}
